apps/client/ui/Slider/Slider.tsx

```diff
@@ -4,17 +4,29 @@ import Slider from 'rc-slider'
 import clsx from 'clsx'
 import './Slider.css'
 
-type CustomSliderProps = Omit<ComponentPropsWithoutRef<'input'>, 'value' | 'onChange'> & {
-  range?: boolean
+type CustomSliderProps = Omit<
+  ComponentPropsWithoutRef<'input'>,
+  'value' | 'onChange'
+> & {
   min?: number
   max?: number
   step?: number
-  defaultValue?: number | number[]
   className?: string
   handleRender?: ReactNode
-  value?: number | number[]
-  onChange?: (value: number | number[]) => void
-}
+} & (
+    | {
+        range?: true
```

Comment:
range in `true` case shouldn't be optional. with `?` it covers `undefined` case with `false` version and the condition lose its power 😄